src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -72,40 +73,40 @@ private static void ThrowNotBlittableJsonReaderObjectOperation(object op)
 
         private static void ThrowMissingDocumentId()
         {
-            throw new InvalidDataException($"Missing '{nameof(DocumentId)}' property on 'TimeSeries'");
+            throw new InvalidDataException($"Missing '{nameof(DocumentId)}' property on 'TimeSeriesOperation'");
+        }
+        private static void ThrowMissingName()
```

Comment:
introduce parameters here and merge those two exceptions?
`ThrowMissingProperty(nameof(DocumentId));`

Reply:
done